resolver/src/main/java/org/apache/james/jspf/parser/RFC4408SPF1Parser.java

```diff
@@ -311,7 +310,7 @@ public SPF1Record parse(String spfRecord) throws PermErrorException,
                             TERM_STEP_REGEX_MECHANISM_POS);
 
                     result.getDirectives().add(
-                            new Directive(qualifier, (Mechanism) mech, log.getChildLogger(qualifier+"directive")));
```

Comment:
aren't you losing a logging facility here?

Reply:
The "directive" part here will be highlighted in the FQCN, I don't think we loose anything.